alpha/apps/kaltura/lib/batch2/kFlowHelper.php

```diff
@@ -1048,6 +1048,11 @@ public static function handleCaptureThumbFinished(BatchJob $dbBatchJob, kCapture
 		$thumbAsset->incrementVersion();
 		$thumbAsset->setStatus(thumbAsset::FLAVOR_ASSET_STATUS_READY);
 
+		$thumbParamsOutput = assetParamsOutputPeer::retrieveByAssetId($data->getThumbAssetId());
+		if($thumbParamsOutput->getInterval())
```

Comment:
We cannot assume that if interval is set then output is bif, since we can have other similar types in the future that will not be bif.
Best have TAG on asset - BIF